src/test/unleash.mock.ts

```diff
@@ -3,6 +3,7 @@
 import { Context, Unleash, Variant } from 'unleash-client';
 import { FeatureInterface } from 'unleash-client/lib/feature';
 import { FallbackFunction } from 'unleash-client/lib/helpers';
+// @ts-ignore
 import { UnleashConfig } from 'unleash-client/lib/unleash';
```

Comment:
TS was complaining about this dependency and I'm not sure why

Reply:
yeah, that sounds familiar. could you add a small note to the ignore? Something like:
```
// @ts-ignore The typescript compiler is complaining about this import for some reason. Can't figure out why.
```

Just so that it's clear to people why it's ignored? Maybe even use `ts-expect-error` so we'll remember to remove it when it's fixed?

Reply:
Out of curiosity, what's the actual complaint that TS produces?

Reply:
https://github.com/Unleash/unleash-client-node/pull/491 - type fix in the source repo so we don't need to ignore anymore